Gemfile

```diff
@@ -6,14 +6,14 @@ ruby RUBY_VERSION
 
 gemspec
 
-gem 'grape', case version = ENV['GRAPE_VERSION'] || '>= 1.5.0'
+gem 'grape', case version = ENV.fetch('GRAPE_VERSION', '>= 1.5.0')
```

Comment:
please can you also adopt it to use the newest grape version: `'~> 1.6'`, thanks